packages/viz/src/declarations/deckgl.d.ts

```diff
@@ -0,0 +1,9 @@
+// TODO: Add proper module declaration;
```

Comment:
In the future maybe we could contribute it to https://www.npmjs.com/package/@danmarshall/deckgl-typings

Reply:
Yes, I would have added our declarations to the repo, but as they were working on 8.0 declarations I didn't want to disturb